src/main/java/com/google/sps/data/DummyDataAccess.java

```diff
@@ -202,7 +202,7 @@ public UserAccount getUser(Key datastoreKey) {
   public Collection<MentorshipRequest> getIncomingRequests(UserAccount user) {
     Collection<MentorshipRequest> data = new ArrayList(5);
     for (int i = 0; i < 5; i++) {
-      // data.add(new MentorshipRequest());
+      data.add(new MentorshipRequest(i+1, i+2));
```

Comment:
is this right?
what happens when the toUser and fromUser don't exist in the system?

Reply:
So this was more for testing purposes to verify that the servlet can retrieve and display the connection requests properly. Once we have our interface implemented with datastore queries, we'll update the methods with the appropriate checks to verify users, but since we aren't really storing any users right now, theres currently no way to verify if they exist or not.

I could write some tests out that builds a bunch of users and forms connections requests between some of them, however I'm afraid it might cause me to fall behind and not finish what we need by wednesday. Once I finish the stuff that needs to be done though I'll write those tests up. Feel free to let me know what your thoughts on this.